socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -61,31 +58,59 @@ export default Vue.component("author-bar", {
         isUserAuthentificated: {type: Boolean, required: true},
     },
     data() {
-        return {showProfileBox: false}
+        return {
+            showProfileBox: false,
+            _isUserFollowingAuthor: this.isUserFollowingAuthor,
+        }
     },
     computed: {
+        isUserRemote(){
```

Comment:
I think it would be good to add tests for the computed `isUserRemote`, `showFollowBtn`, `showUnfollowBtn`, `canFollow` and methods `profileBoxTrigger`, `unfollow`, `follow`. They don't look like they need much changing any more as they're behaving as they should 👍